etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json

```diff
@@ -562,7 +562,7 @@
                 "dashLength": 10,
                 "dashes": false,
                 "datasource": "",
-                "description": "The IOPS panel displays the sum of IOPS (read + write) for a given brick path.",
+                "description": "The IOPS panel displays aggregated brick level read and write operations collected using “gluster volume profile info” over a period of time. The IOPS value shown only reflects metrics from volumes that have volume profiling enabled. No data will be shown if volume profiling is not enabled on at least 1 volume.",
```

Comment:
Why this update required at brick level?

Reply:
As asked by @mbukatov on bug :https://bugzilla.redhat.com/show_bug.cgi?id=1587804#c16, these changes are made!

Reply:
Probably need to use :
"The IOPS panel displays aggregated brick level read and write operations
collected using “gluster volume profile info” over a period of time. The IOPS
value shown only reflects metrics from volumes that have volume profiling
enabled. No data will be shown if the brick is not part of a volume for which the volume profiling is enabled."